docs/workflows/logic/jwt-create.md

```diff
@@ -0,0 +1,38 @@
+# JWT Create
+
+The JWT Create node allows a workflow to create a JSON Web Token from a json object and secret and places it on the payload.
+
+![JWT Create Node](/images/workflows/logic/jwt-create-node.png "JWT Create Node")
+
+## Configuration
+
+JWT Create node configuration requires at least a `JSON Template` object, `Secret Template` and an `Algorithm Type`. The JSON template can be given as a path from the payload or a JSON template object, both should resolve to an object. The secret template can either be a payload template path, or the secret itself. The default alogithm type is `HS256`, see the list below for all algorithms.
```

Comment:
Could mention here that it could be a secret or private key

Reply:
It seems like secret key and private key are synonyms in the crypto world. http://searchsecurity.techtarget.com/definition/private-key

I'll add key after Secret every where and I think that should help.